runtime/Cargo.toml

```diff
@@ -21,6 +21,7 @@ serde = { workspace = true }
 serde_json = { workspace = true }
 strfmt = { workspace = true }
 thiserror = { workspace = true }
+tracing = { workspace = true }
```

Comment:
Since no tracing is setup in the runtimes, we should replace those log lines with `println`s instead

Reply:
Done: 8647dd3a352747ce042cc75b0e9ac9a44e879364